src/index.js

```diff
@@ -217,3 +266,6 @@ if (!module.parent) {
 }
 
 exports.calculateTags = calculateTags;
+exports.calculateTagsFromList = calculateTagsFromList;
+// run is exported so it can be tested
+exports.testExports = { run };
```

Comment:
Nice call to name this testExports rather than `run` to clarify it's not meant to be used for other purposes!